src/routes/(app)/components/LinksList/LinksTable.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { page } from '$app/stores';
+  import TableItem from './TableItem.svelte';
+  const date = new Date();
+
+  console.log($page.data.user.links);
+</script>
+
+<div class="flex flex-col mt-6">
+  <div class="overflow-x-auto rounded-lg">
+    <div class="inline-block min-w-full align-middle">
+      <div class="overflow-hidden shadow sm:rounded-lg">
+        <table class="min-w-full divide-y divide-gray-200 dark:divide-gray-600">
+          <thead class="bg-gray-50 dark:bg-gray-700">
+            <tr>
+              <th
+                scope="col"
+                class="p-4 text-xs font-medium tracking-wider text-left text-gray-500 uppercase dark:text-white"
+              >
+                Hash/ID
+              </th>
+              <th
+                scope="col"
+                class="p-4 text-xs font-medium tracking-wider text-left text-gray-500 uppercase dark:text-white"
+              >
+                URL
+              </th>
```

Comment:
I think we could have a "Created At" column now.
This PR: https://github.com/whizzes/linx/pull/33 exposes the value so we can consume it here.

Reply:
sure!